non-msrv-tests/tests/ui/use_case.stderr

```diff
@@ -1,57 +1,53 @@
 error: derive-where doesn't support empty items, as this can already be handled by standard `#[derive(..)]`
  --> tests/ui/use_case.rs:4:1
   |
-4 | / #[derive_where(Clone)]
-5 | | struct StructEmpty {}
-  | |_____________________^
+4 | struct StructEmpty {}
+  | ^^^^^^^^^^^^^^^^^^^^^
 
 error: derive-where doesn't support empty items, as this can already be handled by standard `#[derive(..)]`
- --> tests/ui/use_case.rs:8:1
+ --> tests/ui/use_case.rs:7:1
   |
-8 | / #[derive_where(Clone)]
-9 | | struct TupleEmpty();
-  | |____________________^
+7 | struct TupleEmpty();
+  | ^^^^^^^^^^^^^^^^^^^^
 
 error: derive-where doesn't support empty items, as this can already be handled by standard `#[derive(..)]`
-  --> tests/ui/use_case.rs:12:1
+  --> tests/ui/use_case.rs:10:1
    |
-12 | / #[derive_where(Clone)]
-13 | | struct Unit;
-   | |____________^
+10 | struct Unit;
+   | ^^^^^^^^^^^^
 
 error: derive-where doesn't support empty items, as this can already be handled by standard `#[derive(..)]`
-  --> tests/ui/use_case.rs:16:1
+  --> tests/ui/use_case.rs:13:1
    |
-16 | / #[derive_where(Clone)]
-17 | | struct UnionEmpty {}
-   | |____________________^
+13 | struct UnionEmpty {}
+   | ^^^^^^^^^^^^^^^^^^^^
 
```

Comment:
not identical to the original but shows the item now

Reply:
I think that's fine.